engine/class_modules/priest/sc_priest_discipline.cpp

```diff
@@ -408,7 +510,7 @@ void priest_t::create_buffs_discipline()
   buffs.harsh_discipline = make_buff( this, "harsh_discipline", talents.discipline.harsh_discipline )
                                ->set_max_stack( talents.discipline.harsh_discipline.enabled()
                                                     ? talents.discipline.harsh_discipline->effectN( 1 ).base_value()
-                                                    : 999 )
+                                                    : 0 )
```

Comment:
you cannot set this to 0. If the talent is not found you make a buff with 0 stacks, which simc will complain about on every sim run (even shadow priest)

Reply:
Reverted to 999.